iosApp/iosApp/Pages/StopDetails/StopDetailsFilteredDepartureDetails.swift

```diff
@@ -119,7 +119,6 @@ struct StopDetailsFilteredDepartureDetails: View {
                             icon: routeSlashIcon(routeType)
                         )
                         .accessibilityHeading(.h4)
-                        .accessibilityFocused($selectedDepartureFocus, equals: cardFocusId)
```

Comment:
[Discussed in slack here](https://mbta.slack.com/archives/C062QNAJZ2M/p1738083419041429?thread_ts=1738082727.891259&cid=C062QNAJZ2M): It is not a possible state for the cancelled trip card to take focus, so removing from the iOS side & not porting to android.